dicomImport/matRad_importDicom.m

```diff
@@ -1,13 +1,16 @@
-function [ct, cst, pln, resultGUI] = matRad_importDicom( files )
+function [ct, cst, pln, resultGUI] = matRad_importDicom( files, dicomBool )
```

Comment:
Maybe rename to dicomMetaBool? Not so happy with the name of this variable... Any other suggestions?